src/index.html

```diff
@@ -6,9 +6,69 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="./images/logo.png"
+          alt="Moyo-logo"
+          class="header__logo-pic">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav_list">
+          <li class="header__nav_item">
```

Comment:
let's mix BEM blocks
```suggestion
      <nav class="header__nav nav">
        <ul class="nav__list">
          <li class="nav__item">
```